File: app/src/main/java/com/philkes/pin2pdf/api/pinterest/model/ArticleData.java
package com.philkes.pin2pdf.api.pinterest.model;

import static com.philkes.pin2pdf.Util.toStr;

public class ArticleData {
    private String name;

    public ArticleData() {
    }

    public String getName() {
        return toStr(name);
    }

    public void setName(String name) {
        this.name=toStr(name);
    }
}

File: app/src/main/java/com/philkes/pin2pdf/api/pinterest/model/ArticleData.kt
package com.philkes.pin2pdf.api.pinterest.model;

import org.apache.commons.lang3.StringEscapeUtils;


public class ArticleData {
    private String name;

    public ArticleData() {
    }

    public String getName() {
        return StringEscapeUtils.unescapeHtml3(name);
    }

    public void setName(String name) {
        this.name=StringEscapeUtils.unescapeHtml3(name);
    }
}
